bottles/frontend/ui/about.blp

```diff
@@ -1,6 +1,6 @@
 using Gtk 4.0;
 
-.AdwAboutWindow about_window {
+$AdwAboutWindow about_window {
```

Comment:
Even though the blueprint lsp suggests to make this change, its supposed to be `.AdwAboutWindow` -> `Adw.AboutWindow` including `using Adw 1;` in all files that use Adwaita.